# Gorilla 158 rebase: browser/components/preferences/main.inc.xhtml as built on Firefox 158
# The earlier patches of this file do not apply to pristine 158 (superseded in the policy with
# exclude_reason_158); this patch carries the file from the remaining set to the built tree.

diff --git a/browser/components/preferences/main.inc.xhtml b/browser/components/preferences/main.inc.xhtml
index 71c7725d93b9a784b840bf44e2cca19a3f490307..0cdf7eec4cf14124708bd3a3c35911e653fc6255 100644
--- a/browser/components/preferences/main.inc.xhtml
+++ b/browser/components/preferences/main.inc.xhtml
@@ -104,31 +104,8 @@
             data-l10n-args='{"localeName": "und"}'
             preference="intl.regional_prefs.use_os_locales"/>
 
-  <!-- This Translations UI is the pre-settings-redesign UI and should be removed
-       when getting ready for the settings redesign to fully ship. -->
-  <vbox id="translationsGroup" data-srd-groupid="translations" hidden="true" data-subcategory="translations">
-    <label><html:h2 data-l10n-id="translations-manage-header"/></label>
-    <hbox id="translations-manage-description" align="center">
-      <description flex="1" data-l10n-id="translations-manage-intro-2"/>
-      <button id="translations-manage-settings-button"
-              is="highlightable-button"
-              class="accessory-button"
-              data-l10n-id="translations-manage-settings-button"/>
-    </hbox>
-    <vbox>
-      <html:div id="translations-manage-install-list" tabindex="-1">
-        <hbox class="translations-manage-language">
-          <label data-l10n-id="translations-manage-download-description"></label>
-          <button id="translations-manage-install-all"
-                  data-l10n-id="translations-manage-language-download-all-button"></button>
-          <button id="translations-manage-delete-all"
-                  data-l10n-id="translations-manage-language-remove-all-button"></button>
-        </hbox>
-        <!-- The downloadable languages will be listed here. -->
-      </html:div>
-      <description id="translations-manage-error" hidden="true"></description>
-    </vbox>
-  </vbox>
+  <!-- GORILLA UNLEASHED - PHYSICAL LOCK: no translations (D-157-03): the legacy Translations section
+       (#translationsGroup: language downloads and the translations settings dialog button) is removed. -->
 </groupbox>
 
 <!-- This Translations UI is compatible with the settings redesign -->
